src/styles/partials/project.sass

```diff
@@ -0,0 +1,88 @@
+.project
+    border: 0.5px solid #DADADA 
+    box-sizing: border-box
+    padding-left: 77px
+    position: relative
+    color: $text-black
+    margin-top: 24px
```

Comment:
Change to this:
![image](https://user-images.githubusercontent.com/29755018/101206689-259db500-3678-11eb-8776-a8d5b7f4c37d.png)